admin/css/admin.css

```diff
@@ -427,6 +427,14 @@ body.post-type-volunteer_opp .wp-pointer-arrow {
 }
 
 /**
+* Widget styling
+*/
+.wi-widget-block-label:after {
```

Comment:
This is a really interesting approach. Can you talk us through why you chose to do it via a pseudo-element vs. using `display: block;` for the code?